receiver/prometheusreceiver/metrics_receiver.go

```diff
@@ -51,62 +52,55 @@ func newPrometheusReceiver(logger *zap.Logger, cfg *Config, next consumer.Metric
 
 // Start is the method that starts Prometheus scraping and it
 // is controlled by having previously defined a Configuration using perhaps New.
-func (pr *pReceiver) Start(_ context.Context, host component.Host) error {
-	pr.startOnce.Do(func() {
-		ctx := context.Background()
-		c, cancel := context.WithCancel(ctx)
-		pr.cancel = cancel
-		c = obsreport.ReceiverContext(c, pr.cfg.Name(), "http", pr.cfg.Name())
-		var jobsMap *internal.JobsMap
-		if !pr.cfg.UseStartTimeMetric {
-			jobsMap = internal.NewJobsMap(2 * time.Minute)
+func (r *pReceiver) Start(ctx context.Context, host component.Host) error {
```

Comment:
This was an old pattern in the collector that every component was protecting itself against calling start multiple times. This whole logic can be removed, the collector framework guarantees that start is called only once, and shutdown as well.

I think the whole start can be simplified more.